.editorconfig

```diff
@@ -0,0 +1,32 @@
+root = true
+
+[*]
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
```

Comment:
I would recommend adding `insert_final_newline = true` for all files.  This will stop disputes between vim, Visual Studio, and a number of other editors that have different defaults for that.